packages/type-jsonrpc/src/author.ts

```diff
@@ -22,8 +22,18 @@ const submitExtrinsic: MethodOpt = {
   type: 'Hash'
 };
 
+const extrinsicUpdate: MethodOpt = {
+  isSigned: true,
+  description: 'Subscribe and watch an extrinsic',
+  params: [
+    createParam('metadata', 'subscriber', 'Bytes')
```

Comment:
You should not need the first 2 params, only the last one. (The first 2 are internal for subscriptions)

Reply:
See e.g. `state_storage` or `chain_newHead` (also subscription APIs). Additionally, you need the `subscription` param, once again look at the above 2. (Yes, it is a bit hairy, any cleanups appreciated)